androidApp/src/main/java/com/mbta/tid/mbta_app/android/component/UpcomingTripView.kt

```diff
@@ -93,7 +93,7 @@ fun UpcomingTripView(state: UpcomingTripViewState) {
                     )
                 is TripInstantDisplay.Approaching ->
                     BoldedTripStatus(
-                        text = stringResource(R.string.approaching_abbr),
+                        text = stringResource(R.string.minutes_abbr, 1),
```

Comment:
This is how we handle the Approaching case on iOS - there's no separate string for "1 min" - so it seems reasonable to do the same thing here.